music21/figuredBass/notation.py

```diff
@@ -410,15 +518,34 @@ class Figure(prebase.ProtoM21Object):
             associated with an expanded
             :attr:`~music21.figuredBass.notation.Notation.notationColumn`.
             ''',
+        'hasExtender': '''
+            A bool value that indicates whether an extender is part of the figure.
+            It is set by a keyword argument.
+            ''',
+        'isExtender': '''
+            A bool value that returns true if an extender is part of the figure but no
+            number is given. Pure extender if you will.
+            It is set by evaluating the number and extender arguments.
+            '''
     }
 
-    def __init__(self, number=1, modifierString=None):
+    isExtender: bool
+
+    def __init__(self, number=1, modifierString=None, extender=False):
```

Comment:
change to:

```
def __init__(self, number=1, modifierString=None, *, extender: bool = False):
```
so that extender has to be given as a keyword.  Otherwise people will do this in code:

```
n = notation.Figure(3, '#', True)
```

and I'll be scratching my head about what the True means.